src/main/java/chainOfResponsability/VerifyDate.java

```diff
@@ -0,0 +1,17 @@
+package chainOfResponsability;
+
+public class VerifyDate extends HandlerData {
```

Comment:
this shouldn't be verifiers, but sanitizers, and why do you have a reference on the value on the constructor? the only reference to the values should be through dataSanitize method